internal/backends/winit/winitwindowadapter.rs

```diff
@@ -244,6 +244,7 @@ pub struct WinitWindowAdapter {
     fullscreen: Cell<bool>,
 
     pub(crate) renderer: Box<dyn WinitCompatibleRenderer>,
+    opengl_api: Option<OpenGLApi>,
```

Comment:
Instead of storing that here, maybe it's easier to pass it through to the renderer constructors and leave it there, instead of carrying it around here. Eventually the thing we pass through could be a large option anyway that also has the overall graphics API, not just the GL version. But in scope of this PR, would you be willing to try and see if passing to `new_suspended()` is easier? If it's not easier, then we can go with this way :)

Reply:
Sure, I briefly looked into this but had issues with areas like [this](https://github.com/slint-ui/slint/blob/f8f1d468a0aba1b11e7f9fb71739e46a178d4415/internal/backends/winit/lib.rs#L129-L133) and [this](https://github.com/slint-ui/slint/blob/f8f1d468a0aba1b11e7f9fb71739e46a178d4415/internal/backends/winit/lib.rs#L236-L252) expecting `new*_suspended` to not take any arguments.

I would definitely prefer this approach as well though, I'll think on it a little more.

Reply:
You’re right. The parameter you’re adding is similar to the window attributes, and it’s fine to keep them in sync and pass them around the same way.